.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "2.7"
+  - "3.5"
+install:
+  - "pip install -r requirements.txt"
+  - "pip install -r test-requirements.txt"
+  - "pip install flake8"
+before_script:
+ - flake8 .
```

Comment:
This line is not needed because we are testing with flake8 in tests/test-flake8.sh.